test/parallel/test-cluster-net-send.js

```diff
@@ -1,55 +1,56 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var fork = require('child_process').fork;
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+const net = require('net');
 
 if (process.argv[2] !== 'child') {
   console.error('[%d] master', process.pid);
 
-  var worker = fork(__filename, ['child']);
-  var called = false;
+  const worker = fork(__filename, ['child']);
+  let called = false;
 
-  worker.once('message', function(msg, handle) {
-    assert.equal(msg, 'handle');
+  worker.once('message', common.mustCall((msg, handle) => {
+    assert.strictEqual(msg, 'handle');
     assert.ok(handle);
     worker.send('got');
 
-    handle.on('data', function(data) {
+    handle.on('data', common.mustCall((data) => {
       called = true;
-      assert.equal(data.toString(), 'hello');
-    });
+      assert.strictEqual(data.toString(), 'hello');
+    }));
```

Comment:
Yes, drop the `called` variable and the `process.once('exit', ...)`, move the `common.mustCall()` from the `data` handler to the `end` handler, and do a concatenation in the `data` handler.